### PR TITLE
Update on Get Started documentation page.

### PR DESCRIPTION
**Pull request by TiagoJustino**
Updated the documentation with the correct minimal version for Node and some more few tips on the installation process.

### CODE REVIEW

File: docs/src/documentation/GetStarted.htm [hunk 1]
```diff
@@ -22,6 +22,10 @@
           {'npm config set http-proxy http://\{host\}:\{port\}'}</br>
           {'npm config set https-proxy https://\{host\}:\{port\}'}
           </code></pre>
+          <p>If you find problem on downloading packages through https proxy, try using http protocol in your https-proxy varialbe, as in:</p>
```
Comment (alansouzati) [inline docs/src/documentation/GetStarted.htm]: Can we change it to: `if you find problems`.. and also there is a typo in variable as in `varialbe`

File: docs/src/documentation/GetStarted.htm [hunk 2]
```diff
@@ -29,13 +33,13 @@
         <h4>NPM</h4>
           <ol>
              <li>
-              <p>Install Gulp globally</p>
+              <p>Install Gulp globally (make sure to run as admin).</p>
```
Comment (alansouzati) [inline docs/src/documentation/GetStarted.htm]: can we make it as `make sure to run as an admin`

**Comment by TiagoJustino** [comment]
Thank you for spotting that.

**Comment by alansouzati** [comment]
Thanks Tiago!

**Pull request merged**